app/_src/production/init-containers.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Init Containers
+---
+
+Due to the way how {{site.mesh_product_name}} implements transparent proxying, running Kubernetes init containers with network calls can be a challenge.
+
+### The destination of the network call is outside the mesh
+
+The common pitfall is the idea that we can order init containers so that the mesh init container is run after other init containers.
+However, if those init containers are injected into Pod via webhooks (like Vault init container) there is no guarantee of the order.
+Ordering of init containers is also not a solution when CNI is used, because traffic redirection to the sidecar is applied even before any init container is run.
```

Comment:
```suggestion
Ordering of init containers is also not a solution when the CNI is used, because traffic redirection to the sidecar is applied even before any init container is run.
```